functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database

```

Comment:
Yes, I think we could leave the CircleDB out if we want to keep the original title.